guides/plugins/apps/starter/starter-admin-extension.md

```diff
@@ -0,0 +1,155 @@
+# App-Starter - Create an Admin Extension
+
+In this guide, you will learn how to set up an extension for the administration UI.
+
+{% hint style="info" %}
+When you are using a self-managed Shopware Version, make sure to set the feature flag `FEATURE_NEXT_17950=1` to enable the Admin Extension API.
+{% endhint %}
+
+![An admin notification](../../../../.gitbook/assets/extension-api-notification.png)
+
+## Prerequisites
+
+In order to follow this guide, make sure you're familiar with and meet the following requirements:
+
+ * Basic CLI usage (creating files, directories, running commands)
+ * Installed [shopware-cli](https://sw-cli.fos.gg/) tools
+ * We will use the following libraries / softwares
+    * npm
+    * live-server (small local development live-reloading server)
+
+## Create the App Wrapper
+
+First of all we need to create the app "wrapper", the so-called app manifest. It is just a single XML file with some basic configuration.
+
+### Create manifest file
+
+First of all, we create a new directory that contains our project.
+
+```bash
+mkdir ListingExtension
+```
```

Comment:
I'm not a big fan of the bash commands. Maybe a file tree would be more visual. This also reduces cluttering